xbmc/settings/windows/GUIControlSettings.cpp

```diff
@@ -138,6 +150,21 @@ static bool GetIntegerOptions(SettingConstPtr setting, IntegerSettingOptions& op
     }
   }
 
+  switch (pSettingInt->GetOptionsSort())
+  {
+    case SettingOptionsSort::Ascending:
```

Comment:
> Do not indent `case` and `default`.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#332-switch-case

Reply:
I used the same style as present a few lines above but I can change it here.

Reply:
Done